.agent-os/cache/optimization-helpers.js

```
@@ -0,0 +1,241 @@
+        }
+
+        // Write back to manifest
+        fs.writeFileSync(MANIFEST_PATH, JSON.stringify(manifest, null, 2));
+
+        return {
+            loaded_files: manifest.context_registry.loaded_files.length,
+            loaded_sections: manifest.context_registry.loaded_sections.length,
+            cache_hits: manifest.context_registry.cache_hits
+        };
+    } catch (error) {
+        console.error('Failed to update registry:', error.message);
```

ESLint

[error] 'console' is not defined.

(no-undef)